### PR TITLE
Extend procedures for Google GCE

### PR DESCRIPTION
**Pull request by maximiliankolb**
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.3/Katello 4.5
* [x] Foreman 3.2/Katello 4.4
* [x] Foreman 3.1/Katello 4.3

### CODE REVIEW

**Comment by maximiliankolb** [comment]
Can you please review my PR? @Lennonka 
You can ignore the second commit; this will be dropped eventually.

**Comment by maximiliankolb** [comment]
Rebased to master which allowed me to drop the second temporary commit. :v:

File: guides/common/modules/proc_installing-google-gce-plugin.adoc [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+This allows you to manage and deploy hosts to GCE.
+
+.Procedure
+. Install the Google GCE compute resource provider on your {ProjectServer}:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# {foreman-installer} --enable-foreman-compute-gce
+----
+. Optional: In the {ProjectWebUI}, navigate to *Administer > About* and select the _compute resources_ tab to verify the installation of the Google GCE plugin.
```
Comment (Lennonka) [inline guides/common/modules/proc_installing-google-gce-plugin.adoc:14]: Use bold emphasis to format tab name?
And the capitalization of the tab name is consistent with the GUI?

File: guides/common/modules/proc_adding-google-gce-details-to-a-compute-profile.adoc [hunk 2]
```diff
@@ -1,41 +1,39 @@
...
-. Create a compute profile to use with the GCE compute resource:
+. Create a compute profile to use with the Google GCE compute resource:
 +
 [options="nowrap" subs="+quotes"]
 ----
-# hammer compute-profile create --name _gce_profile_
+# hammer compute-profile create --name _My_GCE_Compute_Profile_
 ----
-
 . Add GCE details to the compute profile.
```
Comment (Lennonka) [inline guides/common/modules/proc_adding-google-gce-details-to-a-compute-profile.adoc:30]: Replace the trailing period with a colon.

File: guides/common/modules/proc_deleting-a-vm-on-google-gce.adoc [hunk 3]
```diff
@@ -0,0 +1,11 @@
+[id="Deleting_a_VM_on_Google_GCE_{context}"]
+= Deleting a VM on Google GCE
+
+You can delete VMs running on Google GCE from within {Project}.
```
Comment (Lennonka) [inline guides/common/modules/proc_deleting-a-vm-on-google-gce.adoc:4]: Maybe `on your {ProjectServer}` would be safer than `from within {Project}` ?
I'm not sure how that would behave in automated translation :))

**Review by Lennonka — CHANGES_REQUESTED** [review]
Very nice! Just a few details.

File: guides/common/modules/proc_adding-google-gce-details-to-a-compute-profile.adoc (hunk 2 above)
Comment (maximiliankolb) [reply guides/common/modules/proc_adding-google-gce-details-to-a-compute-profile.adoc]: Applied.

File: guides/common/modules/proc_deleting-a-vm-on-google-gce.adoc (hunk 3 above)
Comment (maximiliankolb) [reply guides/common/modules/proc_deleting-a-vm-on-google-gce.adoc]: Applied.

File: guides/common/modules/proc_installing-google-gce-plugin.adoc (hunk 1 above)
Comment (maximiliankolb) [reply guides/common/modules/proc_installing-google-gce-plugin.adoc]: Fixed, cheched, and applied!

**Review by maximiliankolb — COMMENTED** [review]
I've added all your suggestions.

**Review by Lennonka — APPROVED** [review]
Ready to fly

**Pull request merged**